app/src/components/features/rules/TestThisRule/components/TestRuleHeader/index.tsx

```diff
@@ -37,6 +38,7 @@ export const TestRuleHeader = () => {
   const { sheetPlacement } = useBottomSheetContext();
 
   const { claimIncentiveRewards } = useIncentiveActions();
+  const isNewUserForIncentivization = useIsNewUserForIncentivization("2024-07-05");
```

Comment:
The release date string literal is being used in every instance of `useIsNewUserForIncentivization`. Can we create a constant for release dates for incentivization and use that instead.

Reply:
If in future any more enhancements are made for new users then we can just add a update or add a new date in the constant.

Reply:
Okay